server/app.js

```diff
@@ -4,36 +4,41 @@ import { join } from "path";
 import cookieParser from "cookie-parser";
 import logger from "morgan";
 import mongoose from "mongoose";
-const dotenv = require('dotenv');
+const busboy = require('connect-busboy');
+const busboyBodyParser = require('busboy-body-parser');
 
 import indexRouter from "./routes/index";
 import pingRouter from "./routes/ping";
-import auth from "./routes/auth";
-import verifyToken from "./routes/verifyToken";
+import authRouter from "./routes/auth";
+const shopIdRouter = require("./routes/shopId");
```

Comment:
can be consistent with how you import? We should use the import statement as opposed to the older require statement

Reply:
Do you have any resource to explain the difference? Every resource I have used for learning uses require for Node. This is the first time I have seen import used in Node (in React I use it regularly of course). I prefer consistency too, and almost changed everything to require.